tests/test_dependencies.py

```diff
@@ -61,3 +62,18 @@ def test_dependencies_target_dir_somewhere_else(tmpdir):
         patched.return_value = "/some/profile/dir"
         result = str(dependencies._dependencies_target_dir(tmpdir))
         assert "/some/profile/dir/python" == result
+
+
+def test_get_python_interpreter_linux():
+    python_interpreter = dependencies._get_python_interpreter()
+    head, tail = os.path.split(python_interpreter)
+    assert tail == "python3"
```

Comment:
Ik zou gewoon `assert python_interpreter.endswith("python3")` ofzo doen. Of "python3 in ..." om python 3.7 ook goed te vinden.